br.com.barros.DevWidgets.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+    Generator for metainfo & .desktop files: 
+    https://www.freedesktop.org/software/appstream/metainfocreator/#/
+-->
+<component type="desktop-application">
+  <id>br.com.barros.DevWidgets</id>
```

Comment:
I am pretty sure that `barros.com.br` is UNRELATED to this. So the app-id doesn't meet the requirements.